### PR TITLE
XMGPLAT-2446 Docker Registry for windows

### PR DESCRIPTION
**Pull request by huutribui**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,8 +16,12 @@ clean:
 	rm -rf bin
 
 .PHONY: build-%
+# Attached .exe tail when making wincred file
+build-wincred: # build-wincred only
+	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-wincred" -o "$(DESTDIR)/docker-credential-wincred.exe" ./wincred/cmd/
+
 build-%: # build, can be one of build-osxkeychain build-pass build-secretservice build-wincred
-	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-$*" -o "$(DESTDIR)/docker-credential-$*" ./$*/cmd/
+	go build -trimpath -ldflags="$(GO_LDFLAGS) -X ${GO_PKG}/credentials.Name=docker-credential-$*" -o "$(DESTDIR)/docker-credential-$*.exe" ./$*/cmd/
```
Comment (wcbastian) [inline Makefile]: The .exe wouldn't be needed here, would it? As this entry would be used for anything other than build-wincred

File: registry/auth-server/src/index.js [hunk 2]
```diff
@@ -5,10 +5,6 @@ const port = 8000;
 app.get("/", (req, res) => {
   console.log("my headers:", req.headers);
```
Comment (wcbastian) [inline registry/auth-server/src/index.js]: is this change intentional? Why is it needed?

File: Makefile (hunk 1 above)
Comment (huutribui) [reply Makefile]: Yes it wouldn't be needed in that line. I removed it
